portal/templates/profile_macros.html

```diff
@@ -462,6 +463,116 @@ <h4 class="text-success registration-label">{{_('registered')}}</h4>
+                        <div id="communicationsContainer" class="tabs-container">
+                            <div class="tab">
+                                <input id="tab-four" name="tabStaffGroup" type="radio" data-validate="false" class="tab-input">
+                                <label for="tab-four" class="tab-label">{{_("Send reset password email to user")}}</label>
+                                <div class="tab-content">
+                                    {{resetPassword(person)}}
+                                </div>
+                            </div>
+                            <div class="tab">
+                                <input id="tab-five" name="tabStaffGroup" type="radio" data-validate="false" class="tab-input">
+                                <label id="lbStaffRegEmail" for="tab-five" class="tab-label">{{_("Send registration email to user")}}</label>
+                                <div class="tab-content">
+                                    {{profileStaffRegistrationEmail(person)}}
+                                </div>
+                            </div>
+                        </div>
+                    {%- else -%}
+                        <div class="flex">
+                            <div id="resetPasswordContainer" class="flex-item">
+                                <p class="text-muted communication-prompt">{{_("Send reset password email to staff")}}</p>
+                                {{resetPassword(person)}}
+                            </div>
+                        </div>
+                    {%- endif -%}
+                </div>
+            </div>
+        </div>
+    {%- endif -%}
+    <script>
+        $(function() {
+            $("#communicationsContainer .tab-label").on("click", function() {
+              $("#communicationsContainer .tab-label").removeClass("active");
+              $(this).addClass("active");
+            });
+        });
+    </script>
+{%- endmacro %}
+{% macro profileEmailLog(person) -%}
+    <div id="emailLogContainer">
+        <div id="emailLogContent"></div>
+        <div id="emailLogMessage" class="error-message"></div>
+    </div>
+    <div class="modal fade" id="emailBodyModal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <div class="modal-header-title">{{ _("Email Message Content") }}</div>
+            </div>
+            <div class="modal-body">
+                <div class="body-content"></div>
+            </div>
+            <div class="modal-footer">
+              <button type="button" class="btn btn-default" data-dismiss="modal">{{ _("Close") }}</button>
+            </div>
+          </div>
+        </div>
+    </div>
+    <script>
```

Comment:
Any chance on moving this to a separate file? :)

Reply:
yeah, it makes sense as the file is bloated now.  I will do this in a separate PR.